docs/openrpc.json

```diff
@@ -1251,6 +1252,19 @@
                             "s": {
                                 "title": "s",
                                 "$ref": "#/components/schemas/uint"
+                            },
+                            "v": {
+                                "title": "v",
+                                "description": "For backwards compatibility",
+                                "oneOf": [
```

Comment:
I noticed the official doc uses `$ref: '#/components/schemas/uint'` both for `yParity` and `v`, while we are using `byte` or `null`. Is this intended?

Reply:
both would work. I am following the existing schema that `yparity` already had, and I think is intentional:

```
"byte": {
                "title": "hex encoded byte",
                "type": "string",
                "pattern": "^0x([0-9,a-f,A-F]?){1,2}$"
            },

```

Reply:
Ok, I digged more and compiled the [`openrpc.json` file from the specs](https://github.com/InternetOfPeers/execution-apis/blob/35fb8b528482ce2b2ca92dad6547eae63f1bdcff/openrpc.json) and, weirdly enough, that `$ref: '#/components/schemas/uint'` is not even the final definition, but an intermediate YAML one.

In the final OpenRPC definition, all those `uint` are essentially [`string` with a pattern like this `^0x([1-9a-f]+[0-9a-f]*|0)$`.](https://github.com/InternetOfPeers/execution-apis/blob/35fb8b528482ce2b2ca92dad6547eae63f1bdcff/openrpc.json#L290)

For the future, we should consider to strictly adhering to the generated `openrpc.json` schema from the Ethereum spec, especially if we plan to build a client from the `openrpc.json` file, just to be sure we will be in sync with other clients.

Reply:
@Neurone we do indeed adhere to the specs from the official Ethereum execution-apis repo.
That's what actually pushed us to remove `v` initially.

Also agreed the intermediate specs can be misleading

Reply:
We should fix some parameters to remain in sync with those specs (i.e., the `yParity` definition is different, I assume due to subsequent updates). I created a dedicated enhancement request #1837